src/elm/Page/Profile.elm

```diff
@@ -971,23 +707,22 @@ jsAddressToMsg addr val =
                     -- TODO: Handle PushSubscription Decode error
```

Comment:
I know it was already here, but can you remove this TODO @amiskov?

This is related to notifications but our context changed a lot since we added this, moving forward we should add a more intelligent notification system on the backend, allowing for emails and push.

For now we can just remove it

Reply:
Done.